### ISSUE TITLE
Nadpisywanie nazwy produktu powiązanego z bazą produktów

### ISSUE THREAD

**Issue by robboh** [allegro/allegro-api#5316]

**Comment by robboh** [comment]
Nie można nadpisać nazwy produktu z bazy produktów pomimo tego , że jest ona niewłaściwa.
Próba wysłania własnej nazwy 
'productSet' =>
                [
                    [
                        'product' =>
                            [
                                'name' =>'nowa nazwa produktu',
                                .... 
                            ],
                    ],
                ],
 nie działa. Czy to zamierzone czy bug?

**Comment by MaciejFrackowiak** [comment]
Tak, nie nadpiszesz nazwy produktu. Jeżeli uważasz, że dany produkt posiada niewłaściwe dane to możesz to zgłosić korzystając z [POST /sale/products/{productId}/change-proposals](https://developer.allegro.pl/documentation/#operation/productChangeProposal) lub poprzez formularz na stronie WWW - https://allegro.pl/offer/product/report/{productId}.

**Comment by robboh** [comment]
Takie rozwiązanie dla kilkumilonowych zasobów produktów z bazy danych Tecdoc jest karkołomne. Tecdoc sam już podaje nazwy produktów, dlatego takie produkty powinny czerpać nazwy z zasobów Tecdoc a nie z podpowiedzi sprzedających. Są one często mylące.  Bałagan jaki występuje w nazewnictwie szczególnie widoczny jest w nazwach tarcz hamulcowych gdzie przy wyszukaniu produktu po ean można dostać np prawidłowo nazwany produkt  "ATE 24.0120-0178.1 Tarcza hamulcowa" w innym wyszukiwaniu: "Tarcza hamulcowa ATE Power Disc 24-0322-0132-1-01" (zapis numeru części nie zgodny z notacją Tecdoc i producenta części), albo "2x ATE Tarcza hamulcowa 24.0124-0177.1 4006633144704"  - co jest w szczególności wprowadzające w błąd odnośnie ilości produktów w ofercie. EAN 4006633144704 wcale nie wskazuje na komplet dwóch tarcz 24.0124-0177.1
Błędne nazwy produktów można edytować w panelu użytkownika już po wystawieniu oferty, ale nie nadpisywać przy wystawianiu przez API, co wydaje się mi szczególnie niezrozumiałe.

**Comment by robboh** [comment]
Inna ciekawostka, to to, że przy zgłoszeniu błędu produktu panel nie puszcza mnie do wysłania zgłoszenia, bo oryginalna (błędna) nazwa produktu jest za długa: "Nazwa nie może mieć więcej niż 50 znaków"
przykład dotyczy produktu z EAN 4006633119337, id produktu: 3ab659b3-fbed-4119-aa03-2e3cee7d0d1a
